test/integration/constants_test.ts

```diff
@@ -36,64 +36,6 @@ describe("Constants exposed by top level exports", () => {
       `${CONSTANTS.MIME_CE_JSON}; charset=${CONSTANTS.CHARSET_DEFAULT}`,
     );
   });
-  describe("V0.3 binary headers constants", () => {
```

Comment:
This change seems separate from the rest of the PR. Can you have a separate PR for this change?